src/datasets/glam.md

```diff
@@ -0,0 +1,115 @@
+# GLAM datasets
+
+## Introduction
+
+The GLAM aggregation tables are useful for getting the SQL view of the data that drives GLAM if more exploration is required. GLAM aims to answer a majority of the "easy" questions of how a probe or metric has changed over time. Exploring the GLAM tables could take that a little farther. If you need to dive deeper or aggregate on a field that isn't included here, consider reading [todo].
+
+The GLAM tables:
+* Are aggregated at the client level, not the submission ping level
+* Provide a set of dimensions for subsets: channel, os, process or ping type
+* Are aggregated by build ID and version
+* For each aggregation, the distribution and percentiles over time are calculated
+
+### Firefox Desktop
+
+* Optional info?
+    * Pre-processed from Main telemetry to intermediate data with one row per client per metric per day, then aggregated normalizing across clients.
+    * Clients daily aggregates analagous to clients daily with:
+        * all metrics aggregated
+        * each scalar includes min, max, average, sum, and count aggregations
+        * each histogram aggregated over all client data per day
+        * each date is further aggregated over the dimensions: channel, os, version, build ID
```

Comment:
Could you turn this into a paragraph with complete sentences? I think that'll flow a little better.

Reply:
This section needs some work. I wanted to share some details on the intermediate tables but I personally have a bit limited knowledge of them, so I took some notes of what I was able to find. And added the "optional" just for me to follow up on that.